sf_stats.Api/sf_stats.Api/appsettings.Development.json

```diff
@@ -29,14 +29,14 @@
       {
         "Name": "MSSqlServer",
         "Args": {
-          "connectionString": "Server=CHILLTOP-W10;Database=SF_Stats;Trusted_Connection=True;",
+          "connectionString": "Server=JOSHLAPTOP\\SQLEXPRESS;Database=SF_Stats;Trusted_Connection=True;",
```

Comment:
Should probably fix this.... The best option is to use the appsetting.local.json so we don't have to worry about this changing.